content/packages.md

```diff
@@ -64,6 +64,22 @@ Learn more about the tidyverse package at <http://tidyverse.tidyverse.org>.
       <a href="http://tibble.tidyverse.org/">Learn more ...</a></p>
     </div>
   </div>
+  <div class="package">
+    <img class="package-image" src="/images/hex-stringr.png"></img>
+    <div class="package-info">
+      <h3><a href="http://stringr.tidyverse.org/">stringr</a></h3>
+      <p>stringr provide a cohesive set of functions designed to make working with strings as easy as possible. It is built on top of stringi, which uses the ICU C library to provide fast, correct implementations of common string manipulations.
+      <a href="http://stringr.tidyverse.org/">Learn more ...</a></p>
+    </div>
+  </div>
+  <div class="package">
+    <img class="package-image" src="/images/hex-forcats.png"></img>
+    <div class="package-info">
+      <h3><a href="http://forcats.tidyverse.org/">forcats</a></h3>
+      <p>forcats package is to provide a suite of useful tools that solve common problems with factors. R uses factors to handle categorical variables, variables that have a fixed and known set of possible values.
```

Comment:
feels like a word is missing at start of sentence

Reply:
🤦‍♀️ derp! The inevitable product of my forcing list agreement is a renegade missing word!